src/ILLink.RoslynAnalyzer/DynamicallyAccessedMembers/DynamicallyAccessedMembersSymbol.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+using ILLink.Shared;
+using Microsoft.CodeAnalysis;
+
+namespace ILLink.RoslynAnalyzer
+{
+	public class DynamicallyAccessedMembersSymbol : ValueWithDynamicallyAccessedMembers
```

Comment:
I would probably call this something like `SymbolValue` or something similarly simple.
Yes - it can support DAM, but that almost an after thought.

Reply:
I like SymbolValue - changed.